stan/math/prim/core/complex_base.hpp

```diff
@@ -204,7 +204,8 @@ class complex_base {
   template <typename U>
   complex_type& operator/=(const std::complex<U>& other) {
     using stan::math::square;
-    value_type sum_sq_im = square(other.real()) + square(other.imag());
+    value_type sum_sq_im
+        = (other.real() * other.real()) + (other.imag() * other.imag());
```

Comment:
Might be worth opening a new issue about the `square` oddities so it gets looked at

Reply:
Yeah I'm going to open a bigger issue about our include orders.